tests/manual_jobs/test_record_job_start.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import patch
```

Comment:
In the future, I think a method this narrow doesn't need its own test file, but it's fine for now